telescope/utils.py

```diff
@@ -324,7 +338,7 @@ async def fetch(self, project: str, name: str) -> List[BugInfo]:
                     buglist = await fetch_json(
                         url, headers={"X-BUGZILLA-API-KEY": config.BUGTRACKER_API_KEY}
                     )
-                except aiohttp.ClientError as e:
+                except Exception as e:
```

Comment:
I don't think fetch can raise anything else than `ClientError`, so if we don't raise anything when `"bugs" is not in buglist`, then #1457 is not fixed by this PR

```py
try:
     buglist = ...
     if "bugs" not in buglist:
         raise ValueError(...)
except (aiohttp.ClientError, ValueError) as e:
     ...
```